benchmarks/spatialdata_benchmark.py

```diff
@@ -0,0 +1,89 @@
+# type: ignore
+
+# Write the benchmarking functions here.
+# See "Writing benchmarks" in the asv docs for more information.
+import spatialdata as sd
+
+from .utils import cluster_blobs
+
+
+class MemorySpatialData:
+    # TODO: see what the memory overhead is e.g. Python interpreter...
+    """Calculate the peak memory usage is for artificial datasets with increasing channels."""
+
+    def peakmem_list(self):
+        sdata: sd.SpatialData = sd.datasets.blobs(n_channels=1)
+        return sdata
+
+    def peakmem_list2(self):
+        sdata: sd.SpatialData = sd.datasets.blobs(n_channels=2)
+        return sdata
+
+
+def timeraw_import_inspect():
```

Comment:
typo: add _ after time

Reply:
it's not, it's different from time (https://asv.readthedocs.io/en/stable/writing_benchmarks.html#raw-timing-benchmarks).